dist/windows/installer.nsi

```diff
@@ -41,6 +41,27 @@ Section $(inst_qbt_req) ;"qBittorrent (required)"
   ; Write the installation path into the registry
   WriteRegStr HKLM "Software\qBittorrent" "InstallLocation" "$INSTDIR"
 
+  ; Register qBittorrent as possible default program for .torrent files and magnet links
+  WriteRegStr HKLM "Software\qBittorrent\Capabilities" "ApplicationDescription" "A BitTorrent client in Qt"
+  WriteRegStr HKLM "Software\qBittorrent\Capabilities" "ApplicationName" "qBittorrent"
+  WriteRegStr HKLM "Software\qBittorrent\Capabilities\FileAssociations" ".torrent" "qBittorrent.File.Torrent"
+  WriteRegStr HKLM "Software\qBittorrent\Capabilities\UrlAssociations" "magnet" "qBittorrent.Url.Magnet"
+  WriteRegStr HKLM "Software\RegisteredApplications" "qBittorrent" "Software\qBittorrent\Capabilities"
+  ; Register qBittorrent ProgIDs
+  WriteRegStr HKLM "Software\Classes\qBittorrent.File.Torrent" "" "Torrent File"
+  WriteRegStr HKLM "Software\Classes\qBittorrent.File.Torrent\DefaultIcon" "" '"$INSTDIR\qbittorrent.exe",1'
+  WriteRegStr HKLM "Software\Classes\qBittorrent.File.Torrent\shell\open\command" "" '"$INSTDIR\qbittorrent.exe" "%1"'
+  WriteRegStr HKLM "Software\Classes\qBittorrent.Url.Magnet" "" "Magnet URI"
+  WriteRegStr HKLM "Software\Classes\qBittorrent.Url.Magnet\DefaultIcon" "" '"$INSTDIR\qbittorrent.exe",1'
+  WriteRegStr HKLM "Software\Classes\qBittorrent.Url.Magnet\shell\open\command" "" '"$INSTDIR\qbittorrent.exe" "%1"'
+
+  WriteRegStr HKLM "Software\Classes\.torrent" "Content Type" "application/x-bittorrent"
+  WriteRegStr HKLM "Software\Classes\magnet" "" "URL:Magnet URI"
+  WriteRegStr HKLM "Software\Classes\magnet" "Content Type" "application/x-magnet"
+  WriteRegStr HKLM "Software\Classes\magnet" "URL Protocol" ""
+
+  System::Call 'Shell32::SHChangeNotify(i ${SHCNE_ASSOCCHANGED}, i ${SHCNF_IDLIST}, i 0, i 0)'
```

Comment:
Just a reminder:
```suggestion
  System::Call 'Shell32::SHChangeNotify(i ${SHCNE_ASSOCCHANGED}, i ${SHCNF_IDLIST}, p 0, p 0)'
```
And other instances too.

Reply:
Done.